src/FixedDataTable.js

```diff
@@ -1056,6 +1056,21 @@ var FixedDataTable = React.createClass({
 
     this._scrollHelper.setViewportHeight(bodyHeight);
 
+    // This calculation is synonymous to Element.scrollTop
+    var scrollTop = firstRowIndex && Math.abs(firstRowOffset - this._scrollHelper.getRowPosition(firstRowIndex));
```

Comment:
Changed to 
`var scrollTop = firstRowIndex ? Math.abs(firstRowOffset - this._scrollHelper.getRowPosition(firstRowIndex)) : 0;`

Was also able to get rid of the following if statement (line 1061) because of this